app/controllers/vm_common.rb

```diff
@@ -1313,7 +1313,7 @@ def replace_right_cell(options = {})
     presenter[:hide_modal] = true
     presenter[:lock_sidebar] = @in_a_form && @edit
 
-    presenter.update(:breadcrumbs, r[:partial => 'layouts/breadcrumbs_new'])
+    presenter.update(:breadcrumbs, r[:partial => 'layouts/breadcrumbs_new']) unless not_refresh_breadcrumbs
```

Comment:
Language issue: this should be `dont_refresh_breadcrumbs`.
But then, negative conditions are frowned upon, it would be better to change to `:refresh_breadcrumbs => false`.